libraries/src/Image/Image.php

```diff
@@ -248,109 +248,110 @@ private static function getOrientationString(int $width, int $height): string
+		if (!\is_array($imageSizes))
 		{
-			$thumbSizes = [$thumbSizes];
+			$imageSizes = [$imageSizes];
 		}
 
-		// Process thumbs
+		// Process images
 		$generated = [];
 
-		if (!empty($thumbSizes))
+		if (!empty($imageSizes))
 		{
-			foreach ($thumbSizes as $thumbSize)
+			foreach ($imageSizes as $imageSize)
 			{
-				// Desired thumbnail size
-				$size = explode('x', strtolower($thumbSize));
+				// Desired image size
+				$size = explode('x', strtolower($imageSize));
 
 				if (\count($size) != 2)
 				{
-					throw new \InvalidArgumentException('Invalid thumb size received: ' . $thumbSize);
+					throw new \InvalidArgumentException('Invalid image size received: ' . $imageSize);
 				}
 
-				$thumbWidth  = $size[0];
-				$thumbHeight = $size[1];
+				$imageWidth  = $size[0];
+				$imageHeight = $size[1];
 
 				switch ($creationMethod)
 				{
 					case self::CROP:
-						$thumb = $this->crop($thumbWidth, $thumbHeight, null, null, true);
+						$image = $this->crop($imageWidth, $imageHeight, null, null, true);
 						break;
 
 					case self::CROP_RESIZE:
-						$thumb = $this->cropResize($thumbWidth, $thumbHeight, true);
+						$image = $this->cropResize($imageWidth, $imageHeight, true);
 						break;
 
 					default:
-						$thumb = $this->resize($thumbWidth, $thumbHeight, true, $creationMethod);
+						$image = $this->resize($imageWidth, $imageHeight, true, $creationMethod);
 						break;
 				}
 
-				// Store the thumb in the results array
-				$generated[] = $thumb;
+				// Store the image in the results array
+				$generated[] = $image;
 			}
 		}
 
 		return $generated;
 	}
 
 	/**
-	 * Method to create thumbnails from the current image and save them to disk. It allows creation by resizing or cropping the original image.
+	 * Method to create different sized versions of current image and save them to disk.
 	 *
-	 * @param   mixed    $thumbSizes      string or array of strings. Example: $thumbSizes = array('150x75','250x150');
-	 * @param   integer  $creationMethod  1-3 resize $scaleMethod | 4 create cropping
-	 * @param   string   $thumbsFolder    destination thumbs folder. null generates a thumbs folder in the image folder
+	 * @param   mixed    $imageSizes      string or array of strings. Example: $imageSizes = array('1200x800','800x600');
+	 * @param   boolean  $thumbs          true to generate thumbs, false to generate responsive images
+	 * @param   integer  $creationMethod  1-3 resize $scaleMethod | 4 create by cropping | 5 resize then crop
+	 * @param   string   $destFolder      destination images folder. null generates a responsive or thumbs folder in the image folder
 	 *
 	 * @return  array
 	 *
 	 * @since   2.5.0
 	 * @throws  \LogicException
 	 * @throws  \InvalidArgumentException
 	 */
-	public function createThumbs($thumbSizes, $creationMethod = self::SCALE_INSIDE, $thumbsFolder = null)
+	public function createMultipleSizes($imageSizes, $creationMethod = self::SCALE_INSIDE, $thumbs = false, $destFolder = null)
```

Comment:
From my perspective, I am not a big fan of providing a destFolder, but I'd say, maybe another person can take a rq opinion here.